tsdb/chunkenc/bstream.go

```diff
@@ -113,6 +113,16 @@ func (b *bstream) writeBits(u uint64, nbits int) {
 	}
 }
 
+// wrapper for the standard library's PutUvarint to make it work
+// with our bstream.
+func (b *bstream) putUvarint(x uint64) {
+	buf := make([]byte, 2)
```

Comment:
nit , optimization for later: is this a heap allocation? we should avoid that for 2 bytes, either put into bstream or have it on the stack .

Reply:
noted, let's leave the optimisations for a later PR as suggested